### DISCUSSION TITLE
Roadmap generator with progress tracker

### DISCUSSION THREAD

**Discussion by codereacher15** [scaleracademy/open-source-projects#97] [Ideas]
Creating  and implementing a roadmap generator with the following features-
- Unique to an individual
- Will take into account the following things
- Current skillset
- Knowledge about existing market and the technologies
- Interest to any particular field (if there is any)
-Generate the path according to all the above factor 
 - It will give the following details-
 - Different options available
 - Minimum time required for each option to take a have  a hold on
 - Difficulty of each branch and what are pre requisites
 Finally there will be a progress tracker to track the daily progress

**Comment by saraswati258** [comment]
Nice idea!

**Comment by sujankumar21** [comment]
Great idea , seems very useful for improving efficiency.
